packages/paste-website/src/components/Heading.tsx

```diff
@@ -1,34 +1,48 @@
 import * as React from 'react';
-import {Heading, HeadingProps} from '@twilio-paste/heading';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Heading} from '@twilio-paste/heading';
+import type {HeadingProps} from '@twilio-paste/heading';
 import {styled, themeGet} from '@twilio-paste/styling-library';
+import {LinkIcon} from '@twilio-paste/icons/esm/LinkIcon';
 import {slugify} from '../utils/RouteUtils';
 
 const StyledAnchorHyperlink = styled.a`
   text-decoration: none;
   margin-left: ${themeGet('space.space30')};
-  color: ${themeGet('textColors.colorTextWeak')};
+  color: ${themeGet('textColors.colorTextIcon')};
 
   &:hover {
-    text-decoration: underline;
+    color: ${themeGet('textColors.colorTextLinkStronger')};
   }
 `;
 
+// const StyledAnchorHyperlink: React.FC = (children) => {
+//   return (
+//     <Text margin-left={themeGet('space.space30')} color={themeGet('textColors.colorTextWeak')} as="a">
+//       {children}
+//     </Text>
+//   );
+// };
+
 const AnchoredHeading: React.FC<HeadingProps> = (props) => {
   // Only generate slugs for headings where children is 'string'
   if (typeof props.children === 'string') {
     const id = slugify(props.children);
 
     return (
       <Heading {...props} id={id}>
-        {props.children}
-        <StyledAnchorHyperlink href={`#${id}`} data-cy={`anchored-heading-${props.as}`}>
-          #
-        </StyledAnchorHyperlink>
+        <Box as="span" display="flex" alignItems="center">
+          {props.children}
+          <StyledAnchorHyperlink data-cy={`anchored-heading-${props.as}`} href={`#${id}`}>
```

Comment:
Right now, for screen readers the heading is really verbose because there is the heading text and then an anchor with the heading text again. Ex: "About Data grid about data grid page anchor"

I'd prefer it if the markup was like:
```
<Box>
  <Heading>{heading text}</Heading>
  <StyledAnchorHyperlink>....</StyledAnchorHyperlink>
</Box>
```

That way the heading and the anchor are completely separate.

@SiTaggart what do you think about this?